docs/resources/database.md

```diff
@@ -38,10 +38,34 @@ resource "snowflake_database" "test2" {
 - **data_retention_time_in_days** (Number)
 - **from_database** (String) Specify a database to create a clone from.
 - **from_replica** (String) Specify a fully-qualified path to a database to create a replica from.
+- **from_replica_config** (Block List, Max: 1) Specifies the fully qualified path to a database for replication purposes. (see [below for nested schema](#nestedblock--from_replica_config))
```

Comment:
so how is this different than from_replica?

Reply:
It's not different, I wanted to add this so it would be more clear for users to understand what the fully qualified path is and removes the need for repeatedly escaping the `"` character. It makes more sense to me than the current implementation but I also wanted to allow backwards compatibility so I did not remove the other one but added a `conflicts_with`